scripts/pipeline/create_smcp.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
I don't think we need create_smcp.sh and remove_smcp.sh in this PR.
The pky test code has an init function which will create a default SMCP. If you create or delete a SMCP in a sh script. You may have conflict with the one created in test execution.